source/Node/DirectoryServices/DirectoryServicesExternalSecurityGroupLocator.cs

```diff
@@ -130,6 +132,22 @@ public DirectoryServicesExternalSecurityGroupLocatorResult GetGroupIdsForUser(st
             return new DirectoryServicesExternalSecurityGroupLocatorResult(groups);
         }
 
+        public ExternalSecurityGroupResult Match(string name, CancellationToken cancellationToken)
```

Comment:
Would some logging be helpful here? I can see customers saying "it dont work" - is there anything here we can log to help us when those kinds of tickets come in?

(the rest of this code that this leverages may have been out in the wild long enough for us not to worry about this...)